.changeset/curly-cows-buy.md

```diff
@@ -0,0 +1,6 @@
+---
+"@jpmorganchase/uitk-lab": minor
+"@jpmorganchase/uitk-theme": minor
+---
+
+Changes to Measured characteristic values; change typography -weight token to -fontWeight
```

Comment:
It might be better to have two small PRs than one big PR for these types of changes?

Reply:
good idea thats probably best, will split :)